client/jetpack-connect/auth-form-header.jsx

```diff
@@ -48,57 +49,20 @@ export class AuthFormHeader extends Component {
 		return 'logged-in';
 	}
 
-	getPartnerSlug() {
-		const { partnerId } = this.props.authQuery;
-
-		switch ( partnerId ) {
-			case 51945:
-			case 51946:
-				return 'dreamhost';
-			case 49615:
-			case 49640:
-				return 'pressable';
-			case 51652: // Clients used for testing.
-				return 'dreamhost';
-			default:
-				return '';
-		}
-	}
-
-	getHeaderImage() {
-		const partnerSlug = this.getPartnerSlug();
-
-		let image = false;
-		switch ( partnerSlug ) {
-			case 'dreamhost':
-				image = '/calypso/images/jetpack/dreamhost-jetpack-logo-group.png';
-				break;
-			case 'pressable':
-				image = '/calypso/images/jetpack/pressable-jetpack-logo-group.png';
-				break;
-		}
-
-		if ( ! image ) {
-			return null;
-		}
-
-		return (
-			<div className="jetpack-connect__auth-form-header-image">
-				<img width={ 128 } height={ 42.5 } src={ image } />
-			</div>
-		);
-	}
-
 	getHeaderText() {
 		const { translate } = this.props;
-		const partnerSlug = this.getPartnerSlug();
+		const { partnerSlug } = this.props.authQuery;
 
 		if ( partnerSlug ) {
 			switch ( partnerSlug ) {
 				case 'dreamhost':
-					return translate( 'In partnership with Dreamhost' );
+					return translate( 'Jetpack, in partnership with Dreamhost' );
 				case 'pressable':
-					return translate( 'In partnership with Pressable' );
+					return translate( 'Jetpack, in partnership with Pressable' );
+				case 'milesweb':
+					return translate( 'Jetpack, in partnership with MilesWeb' );
+				case 'bluehost':
+					return translate( 'Jetpack, in partnership with Bluehost' );
 				default:
 					return translate( 'Completing set up' );
```

Comment:
Should we remove this case for unrecognized partners? It should be OK to fall through to the normal handling below. Any reason we'd be concerned about that?